src/content/general/engine/scripting/readme.md

```diff
@@ -177,6 +177,63 @@ Several control structures implicitly return the value of their final expression
 )
 ```
 
+## Object Type Masks
+Several functions from Halo 2 onwards allow the user to perform actions on all objects of a certain type (e.g. bipeds, vehicles etc) that are currently loaded in a scenario. These functions use long values to determine which object types should be referenced.
+
+| Type Mask <long>| Object Type     |
+| ----------------| --------------- |
+| 1               | biped           |
+| 2               | vehicle         |
+| 4               | weapon          |
+| 8               | equipment       |
+| 1024            | crate           |
+
+These values can be added together to effect multiple object types at once. For example, using the function ```(object_destroy_type_mask 5)``` would destroy all bipeds and weapons loaded on the map. Both 0 and 1039 will destroy all objects of the above listed types. The functions which support this are:
```

Comment:
I think the triple backticks here will render as a block, but it looks like you want inline. Single backticks will do that for you `` `(object_destroy_type_mask 5)` ``

Reply:
Oh actually triple back ticks is causing the page to parse weirdly, which stops it from loading the links below properly. Changing them to single backticks should fix this.